rpc/gnmi/mixed-schema.md

```diff
@@ -99,6 +96,35 @@ constrained to the specified origin. Particularly:
   the specified `origin`. To delete contents from multiple origins, a client
   MUST specify multiple paths within the `delete` of the `SetRequest`.
 
+### Special considerations for the use of Origin CLI
+
+The following rules apply to a setRequest containing `origin` CLI.  TODO:
+Special considerations for mixing native schema are currently undefined.
+
+* If a `SetRequest` contains `origin` `cli` and `origin` `openconfig`, the CLI
+  MUST be given precedence over the OpenConfig origin.  This precedence is
+  independent of the order in which origin CLI appears.  This overrides the
+  gNMI specification requirement of ordered path based precedence within a
+  `SetRequest`.  The goal of this precedence is to create deterministic
+  behavior for resolving overlapping configuration.  For setRequests without
+  `origin` CLI, order based precedence still applies.
+* The `update` operation MUST be supported for setRequests containing `origin`
+  `cli` and `origin` `openconfig`.  The `update` operation is compatible with
+  the requirements for `origin` `cli` to take precedence.
+* The `replace` operation for a setRequest containing only `origin` `cli` MUST
```

Comment:
Separate set requests should behave as defined by gNMI.   A setrequest with origin `openconfig` should update or replace the referenced "configuration item(s)" regardless of what setrequest and origin was used in the past.

Reply:
Precedence in this case is determined by use of an `update` or `replace` operation and of course, the ordering of the setrequests.  Each setrequest should be processed as a transaction, separately and atomically.

Reply:
When a gNMI setrequest origin OC sets a new ip address on an interface, the new ip address should take effect regardless of what method was previously used.

Reply:
That would put order/timing dependencies back into the architecture. The IP address leaf in the OC model and the IP address leaf in the native model are two different elements in two different parts of the data tree with two different namespaces. The s/w layers below the management layer need decide how to combine those 2 leafs (native win, error if conflict, etc) and it can't really be order based. Consider a reboot scenario where both leafs were configured by the client. We need either native to take precedence or to error on a conflict to have behavior that is deterministic and repeatable across reboots and normal operations.

Reply:
Agree w/ @jsterne - we seem to be going in circles here I feel... the main point here being that an underlying object always has a specified internal order of precedence - not influenced by way of any API order of operations no matter if combined in the same payload/request or separate.